PWGHF/TableProducer/candidateCreator2Prong.cxx

```diff
@@ -14,13 +14,25 @@
 ///
 /// \author Gian Michele Innocenti <gian.michele.innocenti@cern.ch>, CERN
 /// \author Vít Kučera <vit.kucera@cern.ch>, CERN
+/// \author Pengzhong Lu <pengzhong.lu@cern.ch>, GSI Darmstadt, USTC
+
+#ifndef HomogeneousField
+#define HomogeneousField
+#endif
```

Comment:
The reason to add these three lines is when I try to respect the grouping and sorting of the included head files (move the head files from the KFParticle package to the top), these three lines are needed for compilation, otherwise there will be errors say " could not find the `SetField` function.